swagger_parser/swagger_parser.py

```diff
@@ -6,6 +6,7 @@
 import re
 import six
 import yaml
+import codecs
```

Comment:
Could you keep an alphabetic ordering please? Put `import codec` right before `import datetime`